cfme/markers/serial.py

```diff
@@ -0,0 +1,8 @@
+"""serial: Marker for marking tests as serial (not parallel) tests."""
```

Comment:
Please update this docblock, as I don't think "not parallel" is _quite_ what we want to emphasize here.

My main concern would be with "not parallel" being interpreted as the marked tests would not be collected during a parallelized py.test session, which is not the case.

The docblock here needs to describe what it's doing in more depth - that during a parallelized session, the marked test functions will execute on a single appliance, grouped at the module scope.

Reply:
Yup, definitely agree, updating now